README.md

```diff
@@ -87,20 +87,60 @@ A user can type in the movie they like and the app will give the users a set of
 
 ## Running the app
 ### 1. Initialize the database 
+#### Acquire the datasets
+The original dataset is found [here](https://github.com/SophonPlus/ChineseNlpCorpus/blob/master/datasets/ez_douban/intro.ipynb). Click the first link directs to the [download site](https://pan.baidu.com/s/1DkN1LmdSMzm_jCBKhbPbig) at Baidu Netdisk. Then it could be downloaded by clicking the download button (the second button near on the top right corner, which is next to the blue button). However, it requires one to have Baidu Netdisk installed and a Baidu account. To facilitate downloading, the raw datasets have already been included in this repo under `data/sample/`. They could also be downloaded via the `src/data_acquisition.py` described below.
+
+To download the raw datasets and write them into the S3 bucket, run:
+
+`python src/data_aquisition.py --local_path=<local_path> --s3_path=<s3_path> download_upload`
+
+By default, `python src/data_aquisition.py download_upload` downloads the three raw datasets from `data/sample/` in this repo to local path `data/sample/` and uploads them to `s3://2021-msia423-fei-lanqi/raw/`. Note that the S3 raw folder already contains the raw datasets. To test the script, one might want to write to a different S3 bucket or a different folder in the default bucket. One could also use the subparser `download` or `upload` to only download the datasets from source or only upload the datasets to S3.
```

Comment:
In addition, I got `boto3.exceptions.S3UploadFailedError: Failed to upload data/sample/movies.csv to 2021-msia423-fei-lanqi/raw/movies.csv: An error occurred (AccessDenied) when calling the PutObject operation: Access Denied` when I was trying to run the command. I was wondering whether it needs more configuration. If that is the case, please add some more instructions in README.md.

Reply:
Hi Zixiao, I tried it using a different s3 bucket folder and it worked for me. Looks like the error showed that you are still uploading the files to my bucket? By default, my s3 access policy only allows you to get object and list object. So make sure you are changing the `s3_path` to your s3 bucket path, e.g. `python src/data_aquisition.py --s3_path=s3://2021-msia423-huang-zixiao/raw_test/  upload`. Note that the argument should be provided before the upload subparser.

Reply:
Yes, I could upload the data to my s3 bucket. Then I don't feel like there's any problem here. What I would suggest is to add one or two sentences in `README.md` to talk about this potential error and the emphasize the necessity of specifying the s3 path, because users are prone to use the default command.

Reply:
Thanks for confirming. I will make this part of the instruction more clear.